### PR TITLE
Correct typos and append three new sections.

### PR DESCRIPTION
**Pull request by tdejneka**
@gwww 
Fixed typos and added: Device Configuration, Examples, and Notes.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/upb.markdown [hunk 1]
```diff
@@ -34,7 +38,7 @@ An event is generated whenever a UPB Link is:
 - fade is stopped
 - blink is started
 
-The event is `upb.link_changed`.
+The event is `upb.scene_changed`.
```
Comment (frenck) [inline source/_integrations/upb.markdown:41]: Could you point me to the code?
As for as I can see, there is only 1 even fired: <https://github.com/home-assistant/core/blob/dev/homeassistant/components/upb/__init__.py#L52>

Constant definition:
<https://github.com/home-assistant/core/blob/dev/homeassistant/components/upb/const.py#L16>

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by gwww** [comment]
Code PR to change the event from `upb.link_changed` to `upb.scene_changed` is being created.

The change is being made because everything else in the integration uses `scene_<x>` except for the event.

File: source/_integrations/upb.markdown (hunk 1 above)
Comment (tdejneka) [reply source/_integrations/upb.markdown:41]: I was informed [here](https://community.home-assistant.io/t/upb-lighting/615/289?u=123) that the code would be changed and that I should create a PR to update the documentation.

(See gwww's comment below.)

**Comment by gwww** [comment]
Code change PR: https://github.com/home-assistant/core/pull/35644

**Comment by gwww** [comment]
https://github.com/home-assistant/core/pull/35644 has been accepted.

**Review by frenck — APPROVED** [review]
Thanks, @tdejneka!

**Pull request merged**